openrtb_ext/request_wrapper.go

```diff
@@ -456,6 +511,19 @@ func (re *RequestExt) SetPrebid(prebid *ExtRequestPrebid) {
 	re.prebidDirty = true
 }
 
+func (re *RequestExt) GetSChain() *ExtRequestPrebidSChainSChain {
```

Comment:
Can we please add a comment here saying that these Schain functions on the request ext are only for backwards compatibility for openrtb 2.4 and shouldn't really be used for anything else. Only source.ext.schain or reques.ext.prebid.schain should be used?